ansible/roles/docs-rs-builder/templates/builder.service

```diff
@@ -0,0 +1,25 @@
+#
+# {{ ansible_managed }}
+#
+
+[Unit]
+Description=The docs.rs Builder
+
+[Service]
+Environment=TMPDIR={{ vars_mountpoint_path }}
+Environment=DOCSRS_PREFIX={{ vars_mountpoint_path }}
+Environment=DOCSRS_RUSTWIDE_WORKSPACE={{ vars_mountpoint_path }}
+Environment=DOCSRS_LOG=debug
+Environment=DOCSRS_DATABASE_URL={{ vars_database_url }}
+Environment=DOCSRS_DOCKER_IMAGE=ghcr.io/rust-lang/crates-build-env/linux-micro
```

Comment:
This is wrong, it needs to be using the full image, not linux-micro.

Reply:
Seems to work with the micro one. Do you know what the difference between the two is?

Reply:
The micro one doesn't support all the crates on docs.rs, only crates that have no C dependencies.